standard-bridge-custom-token/package.json

```diff
@@ -14,9 +14,8 @@
     "hardhat": "^2.4.1"
   },
   "dependencies": {
-    "@eth-optimism/contracts": "^0.4.5",
-    "@eth-optimism/hardhat-ovm": "^0.2.2",
-    "@openzeppelin/contracts": "3.3.0",
+    "@eth-optimism/contracts": "canary",
+    "@openzeppelin/contracts": "^4.3.0",
```

Comment:
This version should be fixed to `@openzeppelin/contracts": "4.3.2`

Reply:
Thank you, fixed.